crates/compiler/test_gen/src/helpers/wasm.rs

```diff
@@ -207,11 +208,12 @@ impl<'a> ImportDispatcher for TestDispatcher<'a> {
                 1 => format!(r#"User crash with message: "{}""#, roc_msg),
                 tag => format!(r#"Got an invald panic tag: "{}""#, tag),
             };
-            panic!("{}", msg)
```

Comment:
I think the test failures may be due to changing these lines. The way our internal tests work, I believe they require just a standard panic here.

Reply:
fixed in https://github.com/roc-lang/roc/compare/100a2df04327f19680e578c0f1bb2b87355b4311..e5eb5717eace1559d73561099e9381630b497634